lib/class-wp-rest-widget-updater-controller.php

```diff
@@ -180,6 +236,46 @@ public function compute_new_widget( $request ) {
 			)
 		);
 	}
+
+	/**
+	 * Returns the new widget instance and the form that represents it.
+	 *
+	 * @since 5.2.0
+	 * @access public
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
+	 */
+	public function compute_new_widget( $request ) {
+		$identifier         = $request->get_param( 'identifier' );
+		$is_callback_widget = $request->get_param( 'is_callback_widget' );
+		if ( null === $is_callback_widget ) {
```

Comment:
Adding to the previous comment, if it was specified as an `args` as of type `boolean`, could we assign the default there such that we never expect it to be any value other than `true` or `false`, and not need to include this logic here?